src/test/resources/application-integration-test.yml

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Would it be better to merge `application-integration-test-gha` into this file and set it as the profile? Do I need to specify a value for `in-kubernetes` in this file?

Reply:
Here is my thought about what the end state should be:
- we have a way to set in-kubernetes in the helm values file. It is set to true by default in workspacemanager helm, so that it is true for dev, alpha, staging, production, and our personal environments. This is missing right now.
- we explicitly set it to false in all application-*test yaml files
- we do not set profiles in the testrunner code. It should not be opinionated about whether it is being run as a local test or in a K8s cluster.
What do you think?

Reply:
1. This is what WSM is doing: https://github.com/broadinstitute/terra-helm/blob/master/charts/workspacemanager/templates/deployment.yaml#L185
2. Everywhere can all be false if helm file seems to be the only place where we deploy WSM into kubernetes?

Reply:
Good ideas. I filed https://broadworkbench.atlassian.net/browse/PF-676 for those.

Reply:
> we have a way to set in-kubernetes in the helm values file. It is set to true by default in workspacemanager helm, so that it is true for dev, alpha, staging, production, and our personal environments. This is missing right now.
This is not missing, this is what WSM is doing now.  https://github.com/broadinstitute/terra-helm/blob/master/charts/workspacemanager/templates/deployment.yaml#L185

Reply:
Oh, great! So then we could set it false in application.yml and be done.

Reply:
@jaycarlton I will leave it to you to either set false there and remove the -integration-test-gha or not.